config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'articles#index'
+  # Route _____ controller#action
+  # get '/articles', to: 'articles#index'
+  # get 'articles/:id', to: "articles#show"
+  resources :articles do
+    resources :comments
```

Comment:
- Why are the `comments` resources inside `articles`?
- Where are all the routes for `articles` and `comments`?
- Do you think using `resources :comments` is the best approach here? Why?

Reply:
1. Because comments belongs to an article, if comments are individuals this could go outside from `articles`
2. The routes are created by the resources method, and that block with comments adds the respective url for comments.
3. Yes if i need to update a comment or show the selected comment. But in this case, i can add only the necessary methods, like this:
```
resources :articles do
  post 'comments', to: 'comments#create'
  delete 'comments/:id' to: 'comments#destroy'
end
```